azurerm/internal/services/network/virtual_hub_resource.go

```diff
@@ -100,13 +101,79 @@ func resourceVirtualHub() *schema.Resource {
 				},
 			},
 
+			"defaultRouteTable": {
```

Comment:
all resources should be snake_case as far as I'm aware
```suggestion
			"default_route_table": {
```